<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmWidget</class>
 <widget class="QWidget" name="AlarmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>55</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>316</width>
    <height>55</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>55</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_delete">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delete alarm</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rsc/resources.qrc">
          <normaloff>:/MultiAlarm/delete.ico</normaloff>:/MultiAlarm/delete.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
        <item>
         <widget class="QLabel" name="m_name">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
        <item>
         <widget class="QLabel" name="m_time">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="m_start">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Start alarm</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rsc/resources.qrc">
          <normaloff>:/MultiAlarm/play.svg</normaloff>:/MultiAlarm/play.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
